src/constants.ts

```diff
@@ -126,3 +126,10 @@ export const cosmeticLabels = {
   [R_DEPARTMENT]: 'Department',
   [R_PROTECTIVE_MARKING]: 'Protective Marking'
 }
+
```

Comment:
Oh, I think we've lost a comment from yesterday - probably my mistake.  Fro the `resourcesWithListPage` const, since it is only used in `ReferenceDataForm`, can we declare it in that file?

(happy to discuss...)

Reply:
Done.